status-network-contracts/docs/rln.md

```diff
@@ -0,0 +1,157 @@
+# RLN (Rate-Limiting Nullifier)
+
+## Overview
+
+The RLN (Rate-Limiting Nullifier) contract is a privacy-preserving identity registry that manages a set of identity
+commitments for the protocol participants. It enables users to register their identities, and allows authorized accounts
+to slash misbehaving users through a commit-reveal scheme that prevents front-running attacks.
```

Comment:
We should probably also mention here that this is used in the context of gas free transactions and spam prevention.